src/components/ColorChoiceDialog.js

```diff
@@ -61,6 +78,11 @@ function ColorChoiceDialog(props) {
               color={green}
               onChangeComplete={(result) => setGreen(result.hex)}
             />
+            <br />
```

Comment:
I don't think this is correct layout. Have you tried viewing the page on mobile screen sizes (you can do this in the Chrome Devtools panel) and seeing where this button goes when the grid collapses?

Also, prefer CSS margin over br tags.

Reply:
Just pushed a fix- lmk if this is okay!